src/main/java/com/ssbb/kanban/controller/project/ProjectController.java

```diff
@@ -26,9 +24,25 @@
 
 	@RequestMapping(value = "project", method = RequestMethod.GET)
 	public String loadProject(ModelMap map, HttpSession session) {
+
 		return "project";
 	}
 
+	@RequestMapping(value = "loadSelectedProject", method = RequestMethod.POST)
+	public String loadSelectedProject(ModelMap map, HttpSession session,
+			Project submittedProject) {
+
+		User user = (User) session.getAttribute(Constants.USER);
+		for (Project userProject : user.getProjects()) {

```

Comment:
I think the logic in the request was good, have merged it in, we will most likely need this at some point. I think here looping through the list of projects isn't so bad since we already have loaded all the projects from the DB, so there's no need to re-retrieve this project from the DB again, I think knowing our DB this might be slow! 

I do however think we could encapsulate this logic elsewhere, maybe something like getUserProject(User user, String name) which will get the project from the user using the logic but it would be in helper class where it can be changed, instead of a controller, what do you think? Maybe we could make a userHelper class? or Project helper class of some sort